sdk/metrics/include/opentelemetry-matlab/sdk/metrics/MeterProviderProxy.h

```diff
@@ -32,17 +33,14 @@ namespace otlpexporter = opentelemetry::exporter::otlp;
 
 
 namespace libmexclass::opentelemetry::sdk {
-class MeterProviderProxy : public libmexclass::proxy::Proxy {
+class MeterProviderProxy : public libmexclass::opentelemetry::MeterProviderProxy {
   public:
-    MeterProviderProxy(nostd::shared_ptr<metrics_api::MeterProvider> mp) : CppMeterProvider(mp) {
-        REGISTER_METHOD(MeterProviderProxy, getMeter);
+    MeterProviderProxy(nostd::shared_ptr<metrics_api::MeterProvider> mp) : libmexclass::opentelemetry::MeterProviderProxy(mp), CppMeterProvider(mp) {

```

Comment:
Calling the base class constructor and assigning directly to CppMeterProvider is redundant. The base class constructor already assigns to CppMeterProvider. I think you can leave out CppMeterProvider(mp) here on this line

Reply:
Originally I left out the CppMeterProvider(mp) from this line, but after merging with Rick's code it caused a crash. I had to add it back in or else the "addMetricReader" would cause a crash. Should I find another way to fix that crash instead?

Reply:
If it is causing a crash, we need to understand why. CppMeterProvider(mp) here should be completely redundant. I think we should remove it, find out if it is really crashing, and if so, determine why.

Reply:
Ok I will look into this, thanks.